src/utils/getErrorCodeFromResponse.ts

```diff
@@ -0,0 +1,34 @@
+import { DOMParser } from "xmldom";
+
+import { fromNullable, none, Option, some, tryCatch } from "fp-ts/lib/Option";
+
+/**
+ * Extract StatusMessage from SAML response
+ *
+ * ie. for <StatusMessage>ErrorCode nr22</StatusMessage>
+ * returns "22"
+ */
+export default function getErrorCodeFromResponse(xml: string): Option<string> {
+  return fromNullable(xml)
```

Comment:
from the type definition of this function, `xml` is a `string`, is never null/undefined - wrong type?

Reply:
same as above 
> I took care about this evaluation and enhanced controls in [a0a719c](https://github.com/teamdigitale/italia-backend/commit/a0a719c87e4b77177b4dcb40203e3f94a774c613)